as-ap-machshift/as_ap_machshift/as_ap_machshift.py

```diff
@@ -41,15 +41,16 @@ def __init__(self, app):
         self.app.add_callback(self.update_pv)
 
     def read(self, reason):
-        """Read IOC pvs acording to main application."""
-        return super().read(reason)
+        value = self.app.read(reason)
+        if value is None:
+            return super().read(reason)
+        return value
 
     def write(self, reason, value):
         """Write IOC pvs acording to main application."""
         if self.app.write(reason, value):
             super().write(reason, value)
-        else:
-            return False
+        return False
```

Comment:
@anacso17, is this correct? you always return `False` for write requests?

Reply:
ops, merge problems. thanks!

Reply:
fixed!